external/Java.Interop

```diff
@@ -1 +1 @@
-Subproject commit a1d3ecc8ba6e67b96ae1c633194b0e78c2ac5c23
+Subproject commit d2b5bc101c6643dac5d377451a7a84d232adf93e
```

Comment:
Not sure we need a java.interop change, you might need to update your submodules'